### ISSUE TITLE
Unable to access calendar via keyboard

### ISSUE THREAD

**Issue by JoelHaslam** [grafana/grafana#39840]
**Steps to reproduce**:
Press tab until the reach the icon that allows you to change the current time range. Select this to open up the menu and tab through the 'Relative time ranges' until your keyboard focus reaches the From & To boxes.

**Actual Result**:
https://user-images.githubusercontent.com/63413067/135433559-8d803bce-a761-4614-9a87-7031f3f40e29.mp4

The calendar appears for a very brief moment, and then disappears. Allowing no time to select a date or even test how the keyboard interacts with the calendar.

I would also expect to be able to close the time range menu by pressing Esc (edit: Which you mentioned in a previous comment and has been raised at https://github.com/grafana/grafana/issues/39806)

**Expected Result**:
You are able to interact with the calendar via the keyboard, ideally being able to use the arrow keys to navigate instead of having to tab through every date.

**Relevant WCAG Criteria:** [2.1 keyboard-accessible](https://www.w3.org/TR/WCAG21/#keyboard-accessible)

**Environment**:
- Grafana version: 8.2.0-beta1
- Data source type & version: Default data used at [play.grafana.org](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1)
- User OS & Browser: Windows & Chrome
- Others:

**Comment by torkelo** [comment]
work being done on this as part of accessibility project: https://github.com/grafana/grafana/pull/39070
